nucliadb_node/nucliadb_node/tests/integration/test_indexing.py

```diff
@@ -26,36 +26,36 @@
 from grpc.aio import AioRpcError  # type: ignore
 from nucliadb_protos.noderesources_pb2 import Resource, Shard, ShardId
 from nucliadb_protos.nodewriter_pb2 import IndexedMessage, IndexMessage, TypeMessage
+from nucliadb_utils.utilities import get_storage
 
 from nucliadb_node import SERVICE_NAME, shadow_shards
-from nucliadb_node.app import App
+from nucliadb_node.pull import Worker
 from nucliadb_node.settings import indexing_settings, settings
 from nucliadb_node.shadow_shards import OperationCode
-from nucliadb_utils.utilities import get_storage
 
 TEST_PARTITION = "111"
 
 
 @pytest.mark.asyncio
-async def test_indexing(sidecar: App, shard: str):
+async def test_indexing(worker, shard: str, reader):
     node = settings.force_host_id
 
     resource = resource_payload(shard)
     index = await create_indexing_message(resource, shard, node)
-    await send_indexing_message(sidecar, index, node)  # type: ignore
+    await send_indexing_message(worker, index, node)  # type: ignore
 
     sipb = ShardId()
     sipb.id = shard
 
-    pbshard: Optional[Shard] = await sidecar.reader.get_shard(sipb)
+    pbshard: Optional[Shard] = await reader.get_shard(sipb)
     if pbshard is not None and pbshard.resources > 0:
         processed = True
     else:
         processed = False
 
-    while processed is False:
+    while processed is False:  # pragma: no cover
```

Comment:
Why do we need the pragma no cover here? Should we skip coverage for the whole tests folder?